src/did.ts

```diff
@@ -300,6 +304,10 @@ export class DID {
     jws.link = compatibleCID
 
     if (this._capability) {
+      const exp = this._capability.p.exp
+      if (exp && Date.parse(exp) < Date.now()) {
+        throw new Error('Capability is expired, cannot create a valid signature')
+      }
```

Comment:
I don't think this is needed since this error would be thrown by `this.createJWS` which is called on line 296.

Reply:
yup! removing, then merging